Utils/Dataflow/060_upload2virtuoso/uploadTTL.sh

```diff
@@ -114,16 +123,19 @@ upload_files () {
         ;;
     esac
 
-    eval "$cmd" || { echo "(ERROR) An error occured while uploading file: $INPUTFILE" >&2; continue; }
-
+    eval "$cmd" || { echo "(ERROR) An error occured while uploading file: $INPUTFILE" >&2; continue; } &
```

Comment:
If we send the process to the background, we have a risk of running too many cURL processes, sending data to the Virtuoso (each of them trying to send the content of a file, which can be big enough).

We should check a number of cURL processes before starting a new one -- or simply not put the process to the background and handle single file at a time.

To ensure EOP is sent even in case of a failure, it is better to send it before checking the exit code of the $cmd evaluation. For example, like this:
```
eval "$cmd"
res=$?    # Saving exit code
echo $EOProcess
# Check exit code and if it is not 0, output log message and continue with the next file
[ $res -ne 0 ] && { echo "(ERROR) An error occured while uploading file: $INPUTFILE" >&2; continue; }
```

By the way, I am not sure that "continue" is required here (as there is nothing after it)...